modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -25,6 +25,17 @@ Change Data Capture does **not** capture any data changes resulting from the use
 See link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#non-tx-log-changes/[Change Data Capture -> Key considerations] for more information.
 ====
 
+[role=label--beta]
+== Parquet file support
+Starting with Neo4j 5.24, Neo4j provides support for the Parquet file format in a public beta version.
+The additional parameter `--input-type [csv|parquet]` was introduced to explicitly tell the importer to use either CSV or Parquet.
+Its value defaults to CSV if it is not defined.
+
+Most of the parameters that can be used to configure the import are also valid for the Parquet format.
+There are indicators in the parameter overview to point out which parameters are supported.
```

Comment:
```suggestion
There are indicators in the parameter overview to point out which parameters are not supported.
```
It seems the indicators below are for parameters that the Parquet importer does not support/ignores, or?

Reply:
Yes, I think you're right.